wordcloud/readme_generator.rb

```diff
@@ -3,23 +3,26 @@ class ReadmeGenerator
   ADDWORD = 'add'
   SHUFFLECLOUD = 'shuffle'
 
-  def initialize(octokit:)
+  def initialize(octokit:, label_list:)
```

Comment:
Nit: Might make more sense to include `label_list` as an argument to generate rather than store it as part of the state. Doesn't make a big difference though